index.html

```diff
@@ -2097,6 +2095,19 @@ <h4>Semantic Interoperability</h4>
 <code>@protected</code> feature in the JSON-LD 1.1 specification.
             </li>
           </ul>
+          <p>
+A human-readable document describing the expected order of values for the
+<code>@context</code> <a>property</a> SHOULD be available. A machine-readable
```

Comment:
This is not testable, and thus should not be normative... thus the lowercase 'should' in the previous text.

Reply:
Fixed in https://github.com/w3c/vc-data-model/pull/546/commits/0221fbb2eb2833505848649e65980e38110d0f55.